meta-directive.html

```diff
@@ -0,0 +1 @@
+<meta ng-repeat="(property, content) in $ctrl.meta" property="{{property}}" content="{{content}}">
```

Comment:
What we have now is passing muster with the facebook debugger.  I'm going to be checking out twitter compatibility later today.